signicat/Issues/kvkTradeRegister/description.xml

```diff
@@ -0,0 +1,463 @@
+			</Description>
+		</Attribute>
+		<Attribute id="restrictionInLegalActionOfficial">
+			<Name>
+				<en>Restriction in legal action of the official</en>
+				<nl>Beperking in rechtshandeling functionaris</nl>
+			</Name>
+			<Description>
+				<en>The restriction in legal action of the official, eg: Provisional rule, placed under guardianship or placed under administration</en>
+				<nl>De beperking in rechtshandeling van de functionaris, zoals: Provisioneel bewind, onder curatele gesteld of onder bewind gesteld</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="suspensionStart">
+			<Name>
+				<en>Start date of suspension</en>
+				<nl>Aanvangsdatum van de schorsing</nl>
+			</Name>
+			<Description>
+				<en>Indicates when the official's suspension started</en>
+				<nl>Geeft aan wanneer de schorsing van de functionaris is gestart</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="suspensionEnd">
+			<Name>
+				<en>End date of suspension</en>
+				<nl>Einddatum van de schorsing</nl>
+			</Name>
+			<Description>
+				<en>Indicates when the official's suspension has ended</en>
+				<nl>Geeft aan wanneer de schorsing van de functionaris beeindigd is</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="emancipationOfMinor">
+			<Name>
+				<en>Emancipation of a minor</en>
+				<nl>Handlichting</nl>
+			</Name>
+			<Description>
+				<en>Emancipation of a minor</en>
+				<nl>Handlichting houdt in dat een minderjarige door toewijzing van een rechter dezelfde bevoegdheden toegekend krijgt als een meerderjarige</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="hasRestriction">
+			<Name>
+				<en>Has restriction</en>
+				<nl>Heeft beperking</nl>
+			</Name>
+			<Description>
+				<en>Indication of whether there is some kind of restriction that affects the determination of an authorization (Yes, No)</en>
+				<nl>Indicatie of er sprake is van enige vorm van beperking, die van invloed is op het bepalen van een bevoegdheid (Ja, Nee)</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="isAuthorized">
+			<Name>
+				<en>Is authorized</en>
+				<nl>Is bevoegd</nl>
+			</Name>
+			<Description>
+				<en>Indication of whether someone is authorized (Yes, No or Not established)</en>
+				<nl>Indicatie of iemand bevoegd is (Ja, Nee of Niet vastgesteld)</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="reason">
+			<Name>
+				<en>Reason for established authorization</en>
+				<nl>Reden van vastgestelde bevoegdheid</nl>
+			</Name>
+			<Description>
+				<en>The reason for (not) being authorized or not being able to determine someone's authorization</en>
+				<nl>De reden voor het niet/wel bevoegd zijn of niet kunnen vastellen van iemands bevoegdheid</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="referenceMoment">
+			<Name>
+				<en>Reference moment</en>
+				<nl>Peilmoment (opvraagmoment uit handelsregister)</nl>
```

Comment:
~(opvraagmoment uit handelsregister)~

Reply:
deze heb ik laten staan, want dat willen we vanuit kvk extra duidelijk hebben